workforce-data-initiative/abacus-tpot
# Refactor/dataframes

#### What does this PR do?
Moves dataframe construction to etp-api: https://github.com/workforce-data-initiative/etp-api
#### Description of Task to be completed?
Remove the `input_db` directory and tests related to this module
#### How should this be manually tested?
- Create a Python virtual environment
- Run `python install -r requirements.txt`
- Run `python setup.py test`
The 2 tests should pass!
